packages/webpack-cms-plugins/HubSpotAutoUploadPlugin.js

```diff
@@ -35,34 +35,29 @@ class HubSpotAutoUploadPlugin {
     setLogger(webpackLogger);
     compiler.hooks.afterEmit.tapPromise(pluginName, async compilation => {
       Object.keys(compilation.assets).forEach(filename => {
-        const asset = compilation.assets[filename];
-        // assets with `emitted = false` haven't changed since the previous compilation
-        if (asset.emitted) {
-          const filepath = asset.existsAt;
-
-          if (!this.autoupload || !isAllowedExtension(filepath)) {
-            return;
-          }
+        const outputPath = compilation.getPath(compilation.compiler.outputPath);
```

Comment:
Is this backwards compatible?

Reply:
Yep, works on 4 and 5

Also, FWIW, no compatibility warnings for the `getPath()` method here https://webpack.js.org/api/compilation-object/#getpath